package.json

```diff
@@ -5,16 +5,20 @@
     "packages/*"
   ],
   "scripts": {
+    "build": "nx run-many --target=build",
```

Comment:
this will run build in parallel and in the correct order for every package in the repo